src/cases/e3sm_io_case_F_scorpio.cpp

```diff
@@ -56,9 +56,9 @@ int e3sm_io_case_F_scorpio::wr_test(e3sm_io_config &cfg,
     return err;
 }
 
-int e3sm_io_case_F_scorpio::rd_test(e3sm_io_config &cfg,
-                                    e3sm_io_decom &decom,
-                                    e3sm_io_driver &driver)
+int e3sm_io_case_F_scorpio::rd_test(e3sm_io_config &cfg E3SM_IO_UNUSED,
```

Comment:
Some files still contain E3SM_IO_UNUSED

Reply:
I can't see it on my side. I think it is just sync delay.

Reply:
A clean PR should not first add E3SM_IO_UNUSED and remove it in a later commit.

Reply:
fixed